core/templates/pages/exploration-editor-page/services/exploration-correctness-feedback.service.spec.ts

```diff
@@ -40,28 +43,31 @@ describe('Exploration Correctness Feedback Service', function() {
         }
       ]
     });
+    httpTestingController = TestBed.inject(HttpTestingController);
+    TestBed.inject(ExplorationDataService);
```

Comment:
Usually, when we inject a service, we assign it a variable, so it can be used later.
This looks problematic if this service is not used in this spec file then it is also not required to be injected.